README.md

```diff
@@ -45,6 +45,12 @@ The name of a module contains the name of the project and its binary version.
 
 Example: `foo_2.13 bar_2.13`
 
+#### - `configs-ignore` (optional)
+
+A list of space-separated names of configurations to ignore. The action will not submit the dependencies of these configurations.
+
+Example of configurations are `compile`, `test`, `scala-tool`, `scala-doc-tool`.
```

Comment:
Is there a way to get the exhaustive list of supported configurations?

Reply:
That's a good question.

Reply:
I often use the `buildStructure` key to get the list of projects and configurations but unfortunately it does not print itself to the shell.

```
sbt:example>show buildStructure
[info] sbt.internal.BuildStructure@2a3e1e53
[success] Total time: 0 s, completed Jan 6, 2023 10:49:10 AM
```

Reply:
Apparently there is no easy way currently: https://stackoverflow.com/questions/25045436/how-to-list-all-configurations-with-their-description